spec/main-process/atom-application.test.js

```diff
@@ -368,6 +370,23 @@ describe('AtomApplication', function () {
     })
   })
 
+  describe('before quitting', function () {
+    it('waits until all the windows have saved their state and then quits', async function () {
```

Comment:
> These are main process tests and use Mocha.

Oh wow, I totally forgot about that. But I remember it now because of the explicit function vs arrow function discussion. 
Thanks for setting the record straight 🙇